src/main/res/values/strings.xml

```diff
@@ -85,23 +85,23 @@
   <string name="name">Name</string>
   <string name="network_error_description">Network error.</string>
   <string name="never">Never</string>
-  <string name="new_updates_available">New updates available</string>
+  <string name="new_updates_available">New application versions available</string>
   <plurals name="new_updates_description_format">
     <item quantity="one">%d new update.</item>
-    <item quantity="other">%d new updates.</item>
+    <item quantity="other">%d applications have newer versions available.</item>
   </plurals>
   <string name="no_description_available_description">No description available.</string>
   <string name="no_proxy">No proxy</string>
   <string name="not_updated_description">This repository has not been used yet. You need to enable it to view
     the applications it provides.</string>
   <string name="number_of_applications">Number of applications</string>
   <string name="ok">OK</string>
-  <string name="open_link_confirm_format">Open link %s?</string>
+  <string name="open_link_confirm_format">Open the link \"%s\"?</string>
   <string name="other">Other</string>
-  <string name="over_wifi">Over Wi-Fi only</string>
-  <string name="parsing_index_error_description">Failed to parse the index file.</string>
+  <string name="over_wifi">Only on Wi-Fi</string>
+  <string name="parsing_index_error_description">Could not parse the index file.</string>
   <string name="password">Password</string>
-  <string name="password_is_not_specified">Password is not specified</string>
+  <string name="password_is_not_specified">Type in the password</string>
```

Comment:
Actually, "Username is not specified" and "Password is not specified" are paired strings. Now you want to replace them with "Username not specified" (idk why to remove "is") and "Type in the password" (which is completely different from the first string). Is it intentional?

Reply:
My bad, fixed now.